src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/sampler/HTTPHCAbstractImpl.java

```diff
@@ -143,6 +141,9 @@
 
     protected HTTPHCAbstractImpl(HTTPSamplerBase testElement) {
         super(testElement);
+        if(!StringUtils.isEmpty(JMeterUtils.getProperty("hc.parameters.file"))) { //$NON-NLS-1$
```

Comment:
Why not load this from a static block instead of loading it for each construction of an object ?